moto/dynamodb2/responses.py

```diff
@@ -359,9 +359,14 @@ def get_item(self):
 
         try:
             item = self.dynamodb_backend.get_item(name, key, projection_expression)
-        except ValueError:
-            er = "com.amazon.coral.validate#ValidationException"
-            return self.error(er, "Validation Exception")
+        except ValueError as e:
+            if str(e) == "No table found":
```

Comment:
Is it possible to throw a custom exception when the table does not exist? I feel like this would be a cleaner approach:
```
try:
...
except TableNotFound:
...
except ValueError:
...
```

Reply:
That's what I did initially, but unfortunately it breaks other tests.  I opted for the simplest solution, though I agree with you that's it's not the most elegant.  Fixing this properly requires fairly major surgery, which I'm pretty hesitant to undertake.

Reply:
...looked at this again.  Decided to just fix the reported issue, without touching anything else.  Let me know what you think.

Reply:
Yeah, I think this makes sense, to fail as early as possible. Thanks!